### PR TITLE
Avoid getting started confusion like …

### PR DESCRIPTION
**Pull request by cvogt**
https://gitter.im/cvogt/cbt?at=57ec271754302caa6f2950b7

### CODE REVIEW

File: stage2/Scaffold.scala [hunk 1]
```diff
@@ -34,16 +34,16 @@ class Build(val context: Context) extends BaseBuild{
     super.dependencies ++ // don't forget super.dependencies here
     Seq(    
       // source dependency
-      DirectoryDependency( projectDirectory ++ "/subProject" )
+      // DirectoryDependency( projectDirectory ++ "/subProject" )
     ) ++
     Resolver( mavenCentral ).bind(
```
Comment (mariussoutier) [inline stage2/Scaffold.scala]: I'd suggest adding a comment here, I think the `Resolver.bind` is a very nice feature in contrast to sbt. I assume it means you can control directly how to resolve your deps, so in a company I'd use Artifactory and cbt wouldn't try to load over the Internet?

File: stage2/Scaffold.scala [hunk 2]
```diff
@@ -34,16 +34,16 @@ class Build(val context: Context) extends BaseBuild{
...
+      // ScalaDependency( "com.lihaoyi", "ammonite-ops", "0.5.5" )
+      // MavenDependency( "com.lihaoyi", "ammonite-ops_2.11", "0.5.5" )
 
       // SBT-style dependencies
-      "com.lihaoyi" %% "ammonite-ops" % "0.5.5"
-      "com.lihaoyi" % "ammonite-ops_2.11" % "0.5.5"
+      // "com.lihaoyi" %% "ammonite-ops" % "0.5.5"
+      // "com.lihaoyi" % "ammonite-ops_2.11" % "0.5.5"
     )
   */
```
Comment (mariussoutier) [inline stage2/Scaffold.scala]: I think the /* .. */ could be removed now.

File: stage2/Scaffold.scala (hunk 1 above)
Comment (cvogt) [inline stage2/Scaffold.scala]: Yep indeed. There is no global list of resolvers, instead you bind individual dependencies (and their transitive dependencies) to individual resolvers.

File: stage2/Scaffold.scala (hunk 2 above)
Comment (cvogt) [inline stage2/Scaffold.scala]: done https://github.com/cvogt/cbt/pull/244

File: stage2/Scaffold.scala (hunk 1 above)
Comment (cvogt) [inline stage2/Scaffold.scala]: added a comment https://github.com/cvogt/cbt/pull/244

**Pull request merged**